lib/timer.py

```diff
@@ -14,11 +14,13 @@
     timer_jobs_state_file,
 )
 from uaclient.jobs.metering import metering_enabled_resources
+from uaclient.jobs.update_contract_info import update_contract_info
 from uaclient.jobs.update_messaging import update_apt_and_motd_messages
 
 LOG = logging.getLogger(__name__)
 UPDATE_MESSAGING_INTERVAL = 21600  # 6 hours
 METERING_INTERVAL = 14400  # 4 hours
+UPDATE_CONTRACT_INFO_INTERVAL = 14400  # 4 hours
```

Comment:
The contract doesn't change very often, so we can probably increase the interval - maybe every 24 hours?

Reply:
Updated